### PR TITLE
Clean up old 'blue' style path, update documentation and samples

### PR DESCRIPTION
**Pull request by dannon**
to refer to plain old 'static' now.

Includes a config rebuild (necessary for the changes here) that picked up a few extra bits that look like they got left out of a previous PR?

### CODE REVIEW

File: doc/source/admin/nginx.md [hunk 1]
```diff
@@ -130,10 +130,6 @@ http {
         }
 
 		# serve framework static content
-		location /static/style {
-			alias $galaxy_root/static/style/blue;
-			expires 24h;
-		}
 		location /static {
```
Comment (nsoranzo) [inline doc/source/admin/nginx.md]: Not connected to this PR, but I noticed that the code here is mixing spaces and tabs: convert all tabs to spaces?

Comment (dannon) [reply doc/source/admin/nginx.md]: @nsoranzo Sure, I'll take a look.

Comment (dannon) [reply doc/source/admin/nginx.md]: (done)

File: doc/source/admin/nginx.md [hunk 2]
```diff
@@ -203,8 +199,8 @@ previous section:
             }
 
             # serve framework static content
-            location /galaxy/static/style {
-                alias $galaxy_root/static/style/blue;
+            location /static {
```
Comment (nsoranzo) [inline doc/source/admin/nginx.md]: Shouldn't this be
```suggestion
            location /galaxy/static {
```

Comment (dannon) [reply doc/source/admin/nginx.md]: Yes it should, good catch.

**Comment by nsoranzo** [comment]
Last one: should also https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy_test/driver/driver_util.py#L462 be updated?

**Comment by dannon** [comment]
@nsoranzo Looks like it; that join foiled my grepping!  I'll make another quick check and fix that up.

**Review by nsoranzo — APPROVED** [review]

**Review by natefoo — APPROVED** [review]
Thanks @dannon!

**Pull request merged**